### ISSUE TITLE
Banned members keep showing up in group member list, at least if recently seen

### ISSUE THREAD

**Issue by php4fan** [telegramdesktop/tdesktop#26940]
### Steps to reproduce

1. In a supergroup where you are admin, go to the list of member
2. Pick one in particular and click on it to see their profile
3. Have someone else ban them from the group (in my tests it was a bot who banned the user) (I'm guessing the issue wouldn't reproduce if you banned the user yourself)
4. Go back to see the list of member, and search for the user that has just been banned

### Expected behaviour

The user shouldn't show up anymore.
If there's some sort of caching, there should be a "refresh" option.

Even if the user initially shows up because the list was cached very few seconds ago, at some point, for example when clicking on the user name in the list, something should trigger a refresh and I should see that the user is no longer a member of the group

### Actual behaviour

The user still shows up and there is no indication whatsoever that they have been banned. I can still search and it shows up in results, I can click on them and see their profile, everything looks like the user is still a member.
Additionally, there is no "refresh" or "update" option to make sure that the list of members I'm seeing is up to date.

If I go to the "recent actions" I can see that the user was banned.

And the most ridiculous thing: I can go to Manage Group -> Permissions -> Removed users, and the user is there (so far so good), but **even after this**, if I go back to the list of members, the user still shows up and also still shows up in search results.

Unsurprisingly, if I quit Telegram Desktop and relaunch it, the list will finally be refreshed and the user will disappear from it. But that can't be the only way.

### Operating system

Manjaro Linux

### Version of Telegram Desktop

4.9.7

### Installation source

Other (unofficial) source

**Comment by Aokromes** [comment]
yeah i confirm this bug, specially if banned by bots.

**Comment by weirdvic** [comment]
I can confirm this issue for Telegram Desktop for Windows.
The users who were banned by calling `banChatMember` as a bot, are still showing in chat.

**Comment by php4fan** [comment]
Again the stupid bot that wants to close issues _because_ they are not fixed.

**Comment by php4fan** [comment]
Again the stupid bot as usual...

**Comment by Aokromes** [comment]
yeah it's very typical on groups with a lot of spammers and bots.
